manager/app/src/main/java/me/weishu/kernelsu/ui/screen/Home.kt

```diff
@@ -200,6 +200,26 @@ private fun InfoCard() {
             val contents = StringBuilder()
             val uname = Os.uname()
 
+            val GetSELinuxStatus = Runtime.getRuntime().exec("getenforce")
```

Comment:
Please correct variable name

Reply:
Please follow the java naming convention, do not start with an uppercase letter

Reply:
Put this to a single method, because we may change it implementation in the future(use libselinux instead)

Reply:
ok, I will do

Reply:
has made it a method and change variable to start with a lowercase letter